### ISSUE TITLE
Group Home > Settings: Braille settings no longer appear

### ISSUE THREAD

**Issue by bree-z** [cuny-academic-commons/commons-in-a-box#271]
Hi Boone, 

In Group Home > Settings > Settings, Braille settings no longer appear, even when the Community Features Braille plugin is activated.

<img width="984" alt="Screen Shot 2020-08-13 at 9 57 18 AM" src="https://user-images.githubusercontent.com/7189824/90144434-cde7e080-dd4c-11ea-9d8d-5e8c35130821.png">

Thanks!

**Comment by boonebgorges** [comment]
The group settings tab appears only if our internal utilities determine that the external Braille server is up and running. In our case, it's not - I believe that our original test server was hosted on an AWS instance that was paid for by Matt at the GC, but it's since been retired because it was unused. See https://redmine.gc.cuny.edu/issues/11881. I bypassed this internal health check and confirmed that, when the server is up (or "up", in this case where I faked it), the settings appear as expected.

It's not feasible to set up a test Braille instance at this moment, so I think you'll have to take my word for it that it's working :-D
